src/Careerbuilder.php

```diff
@@ -180,6 +183,52 @@ function ($value, $key) use (&$queryString) {
     }
 
     /**
+     * Get min and max salary numbers from string
+     *
+     * @return array
+     */
+    public function parseSalariesFromString($input = null)
```

Comment:
True. I'm an idiot at regex, so my first battle was making that work. Seems like it does, so I'll jump back in later this week and break it up into multiple methods to see if that helps.